lib/beaker/shared/options_resolver.rb

```diff
@@ -0,0 +1,21 @@
+module Beaker
+  module Shared
+    # Methods for parsing options.
+    # - determine if a mode should be run in parallel
+    module OptionsResolver
+      def run_in_parallel?(opts, options, mode)
```

Comment:
We should probably have some yard docs in repo for this method; checkout [scp example](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L358) for a pretty good yarddoc precedent.